runtimes/pythonrt/dockerclient.go

```diff
@@ -30,31 +30,35 @@ const (
 )
 
 type dockerClient struct {
-	client          *client.Client
-	activeRunnerIDs map[string]struct{}
-	allRunnerIDs    map[string]struct{}
-	mu              *sync.Mutex
-	runnerLabels    map[string]string
-	logBuildProcess bool
-	logRunner       bool
-	logger          *zap.Logger
+	client                     *client.Client
+	activeRunnerIDs            map[string]struct{}
+	allRunnerIDs               map[string]struct{}
+	mu                         *sync.Mutex
+	runnerLabels               map[string]string
+	logBuildProcess            bool
+	logRunner                  bool
+	logger                     *zap.Logger
+	maxMemoryBytesPerContainer int64
+	maxNanoCPUPerContainer     int64
```

Comment:
you can just store the config here, and then you don't need to copy all the stuff. it's dry